### ISSUE TITLE
Missing group, can you add it

### ISSUE THREAD

**Issue by 0x0021** [rrva/openconnectui2#3]
Missing group, can you add it

**Comment by rrva** [comment]
Please explain a bit more. Do you mean support for `--authgroup` ? 

```
Select GROUP from authentication dropdown (may be known
                                  as "realm", "domain", "gateway"; protocol-dependent)
```

**Comment by stsergo** [comment]
Yes, sure. For example, on my vpn gateway where are a lot of auth groups:
<img width="314" alt="image" src="https://github.com/rrva/openconnectui2/assets/18488041/eafabed8-6ab2-4d6a-bee7-1dca40c16d74">

**Comment by rrva** [comment]
New release out now version 1.783 which should fix this. Add --authgroup foo in the custom args section

**Comment by 0x0021** [comment]
> 新版本现已发布 1.783 版，应该可以解决这个问题。在自定义 args 部分添加 --authgroup foo

![image](https://github.com/rrva/openconnectui2/assets/3069412/0e2a30a1-01e8-4af8-9ace-4eecba764776)

There is a problem with the character limit range. My group here contains Chinese characters.

**Comment by rrva** [comment]
Will look into a better solution. If I need to whitelist characters, which system locale are you running?

**Comment by 0x0021** [comment]
> Will look into a better solution. If I need to whitelist characters, which system locale are you running?将研究更好的解决方案。如果我需要将字符列入白名单，您正在运行哪种系统区域设置？

![image](https://github.com/rrva/openconnectui2/assets/3069412/c2372629-bd53-4c5b-bd9c-b1b7c4dac340)
